Mobile Buy SDK/Mobile Buy SDK Tests/BUYRequestOperationTests.m

```diff
@@ -277,7 +277,7 @@ - (void)testCancellationDuringExecution
 		[operation cancel];
 	}];
 	
-	[self waitForExpectationsWithTimeout:4.0 handler:^(NSError *error) {}];
+	[self waitForExpectationsWithTimeout:5.0 handler:^(NSError *error) {}];
```

Comment:
Just out of curiosity, is this intentional and / or does it fix the BuildKit crash we've been experiencing?